.github/workflows/js-lint.yml

```diff
@@ -39,16 +39,13 @@ jobs:
     timeout-minutes: 20
     steps:
       - uses: styfle/cancel-workflow-action@0.8.0
-      - uses: actions/checkout@v2
-      - name: Read .nvmrc
-        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
-        id: nvm
+      - uses: actions/checkout@v3
       - name: Setup Node.js (via .nvmrc)
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+          node-version-file: '.nvmrc'
       - name: Cache `node_modules`
-        uses: actions/cache@v1
+        uses: actions/cache@v3
```

Comment:
This is not needed anymore, we can just use `cache: npm` property for the `actions/setup-node` action and it will take care of cache.

https://github.com/actions/setup-node#caching-global-packages-data

Reply:
Thank you for the feedback. I have updated workflow as requested can you please re-review?